intel_extension_for_transformers/llm/runtime/graph/scripts/load_peft_and_merge.py

```diff
@@ -0,0 +1,59 @@
+#  Copyright (c) 2023 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import argparse
+from peft import PeftModel
```

Comment:
do we already have this in requirements？

https://huggingface.co/docs/transformers/main/peft#setup

Reply:
I see it under nerual_chat or chatbot requirements, could you add to the root requirements?
https://github.com/intel/intel-extension-for-transformers/blob/main/requirements.txt

Reply:
perhaps put it in here https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/requirements.txt would be better?